std/range/package.d

```diff
@@ -11786,12 +11734,10 @@ public:
         {
             return (*_range).length;
         }
-
```

Comment:
Unrelated!

Reply:
cool

Reply:
ah I noticed it was merged... guess it'll stay, yay